molgenis-api-permissions/src/main/java/org/molgenis/api/permissions/model/response/PermissionResponse.java

```diff
@@ -0,0 +1,33 @@
+package org.molgenis.api.permissions.model.response;
+
+import com.google.auto.value.AutoValue;
+import java.util.Set;
+import javax.annotation.Nullable;
+import org.molgenis.util.AutoGson;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_PermissionResponse.class)
+@SuppressWarnings(
+    "squid:S1610") // Abstract classes without fields should be converted to interfaces
+public abstract class PermissionResponse {
+
+  @Nullable
```

Comment:
@nullable -> @nullable @checkfornull so that Sonar checks are enabled

Reply:
commented line not changed, so not marked as outdated by github